getting-started/developers-guide.md

```diff
@@ -15,7 +15,7 @@ layout: getting-started
   - Go
     - [with full language support](https://github.com/golang/go/wiki/WebAssembly#getting-started)
     - [targeting minimal size](https://tinygo.org/docs/guides/webassembly/)
-  - [Kotlin](https://kotlinlang.org/docs/reference/native-overview.html)
+  - [Kotlin](https://kotlinlang.org/docs/whatsnew-eap.html#new-kotlin-wasm-target)
```

Comment:
@bashor does it look good to you?

Reply:
Thanks! Looks good, but https://kotl.in/wasm would be better, so later, we can change the redirect target :)

Reply:
Sure, wasn't aware of the redirect.